### PR TITLE
Shahrizal/lookback phase1

### PR DESCRIPTION
**Pull request by shahrizal-binary**

### CODE REVIEW

File: cpanfile [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+requires 'perl', '5.014000';
+requires 'Math::CDF';
+
+on configure => sub {
+    requires 'ExtUtils::MakeMaker';
+};
+
+on test => sub {
+    requires 'Test::Exception';
+    requires 'Test::FailWarnings';
```
Comment (basiliscos) [inline cpanfile]: `Test::Warnings`

File: t/benchmark.t [hunk 2]
```diff
@@ -0,0 +1,117 @@
+#!/etc/rmg/bin/perl
+
+use strict;
+use warnings;
+
+use Test::Most;
+use Test::Warn;
```
Comment (basiliscos) [inline t/benchmark.t]: check `Test::Warnings`

File: t/benchmark.t [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+use Test::Exception;
```
Comment (basiliscos) [inline t/benchmark.t]: I don't see usage of that

File: t/benchmark.t [hunk 4]
```diff
@@ -0,0 +1,117 @@
+#!/etc/rmg/bin/perl
+
+use strict;
+use warnings;
+
+use Test::Most;
```
Comment (basiliscos) [inline t/benchmark.t:6]: Considerr using `Test::More`

Comment (basiliscos) [reply t/benchmark.t:6]: Oh, I see you use `Test::Most`, ok to keep then

File: t/delta.t [hunk 5]
```diff
@@ -0,0 +1,114 @@
+#!/etc/rmg/bin/perl
+
+use strict;
+use warnings;
+
+use Test::Most;
+use Test::Warn;
+use Test::Exception;
```
Comment (basiliscos) [inline t/delta.t]: see above

File: lib/Math/Business/LookBacks.pm [hunk 6]
```diff
@@ -0,0 +1,277 @@
+package Math::Business::LookBacks;
```
Comment (junbon-deriv) [inline lib/Math/Business/LookBacks.pm]: I see a mixed of naming. In contract category definition it is called `lookback` and here it is `lookbacks`. Choose one. I think `lookback` is more correct.

**Pull request merged**